src/Halcyon/Processors/SectionParser.php

```diff
@@ -182,9 +184,9 @@ public static function parseOffset($content)
      * Returns the line number of a found instance of CMS object section separator (==).
      * @param string $content Object content
      * @param int $instance Which instance to look for
-     * @return int The line number the instance was found.
+     * @return int|null The line number the instance was found.
      */
-    private static function calculateLinePosition($content, $instance = 1)
+    public static function calculateLinePosition($content, $instance = 1)
```

Comment:
Why was this changed?

Reply:
PHPStan reports this as unsafe private static call, since it's being called statically within the class. Realistically, I think this should probably be changed to a singleton.